src/loader/implementation.ml

```diff
@@ -503,22 +503,30 @@ let of_cmt (source_id : Odoc_model.Paths.Identifier.SourcePage.t)
 
   (uid_to_id, postprocess_poses source_id vs uid_to_id uid_to_loc)
 
-let read_cmt_infos source_id_opt id cmt_info =
+let read_cmt_infos source_id_opt id cmt_info ~count_occurrences =
   match Odoc_model.Compat.shape_of_cmt_infos cmt_info with
   | Some shape -> (
       let uid_to_loc = cmt_info.cmt_uid_to_loc in
-      match (source_id_opt, cmt_info.cmt_annots) with
-      | Some source_id, Implementation impl ->
-          let map, source_infos =
-            of_cmt source_id id impl uid_to_loc
-          in
+      match (source_id_opt, count_occurrences, cmt_info.cmt_annots) with
+      | Some source_id, _, Implementation impl ->
+          let map, source_infos = of_cmt source_id id impl uid_to_loc in
+          let occ_infos = Occurrences.of_cmt impl in
```

Comment:
We don't check the value of `count_occurrences` here?

Reply:
No, because when we render the source, we anyway want to collect the occurrences information, to render links from source to documentation!

Reply:
Alright, makes sense.